crates/dids/src/method/dht/bep44.rs

```diff
@@ -0,0 +1,233 @@
+use std::{
+    io::Cursor,
+    time::{SystemTime, SystemTimeError, UNIX_EPOCH},
+};
+
+use byteorder::{BigEndian, ReadBytesExt, WriteBytesExt};
+use crypto::CryptoError;
+use keys::key::{KeyError, PublicKey};
+
+const MIN_MESSAGE_LEN: usize = 72;
+const MAX_MESSAGE_LEN: usize = 1072;
```

Comment:
Might want to split this into something like "max V length" since the sign and seq are separate and we need to enforce that v is no longer than 1000b

Reply:
I added a const `MAX_V_LEN` and some documentation to keep it more clear. We need to do the check on `MIN_MESSAGE_LEN` at the beginner of `decode()` regardless so we don't try to access an index out of bounds.